nilearn/decomposition/canica.py

```diff
@@ -18,7 +18,8 @@
 
 @fill_doc
 class CanICA(_MultiPCA):
-    """Perform Canonical Independent Component Analysis [1]_ [2]_.
+    """Perform :term:`Canonical Independent Component Analysis<CanICA>`  \
+    :footcite:`Varoquaux2010c` :footcite:`Varoquaux2010d`.
```

Comment:
```suggestion
    """Perform :term:`Canonical Independent Component Analysis<CanICA>`.
    
    See :footcite:`Varoquaux2010c` and :footcite:`Varoquaux2010d`.
```

A quick fix for the errors in https://github.com/nilearn/nilearn/pull/3558#issuecomment-1455720506
Seems to be a problem on Sphinx side. The links in the API of the functions work but render incorrectly in the summary page: 
![Screenshot from 2023-03-17 12-29-31](https://user-images.githubusercontent.com/63292494/225893268-66472667-8d25-4bf0-ba3e-302c6979d31c.png)